### PR TITLE
chore: add lab for chat bot(JS only)

### LINKED ISSUE #412: chore: add lab for chat bot(JS only)
The first few labs in the JS section will be JavaScript only and short.
The preceding workshops will also be short and only teach a few concepts.
This is to ensure the campers really grasp the fundamentals before moving onto harder concepts.

Here is the first JS workshop:
https://github.com/freeCodeCamp/CurriculumExpansion/pull/411

We need a lab that will immediately following that.

Here is what the campers will know so far:

- let
- const
- assignment
- reassignment
- basic string usage
- string concatenation using + operator only.
- console.log
- camelCase 

My suggestion would be to have campers create something similar to the workshop in this lab.
It could be another bot that talks to the user. 

Maybe the bot will tell the user facts about freeCodeCamp
Maybe the bot will tell the user facts about JavaScript
Maybe the bot will tell a story instead.
Or maybe the bot will just have another random conversation with the user like in the workshop.

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by larymak**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #412

### CODE REVIEW

File: frontend-cert/js-projects/js-chat-bot/script.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+console.log("Hello! I'm JSBot, your JavaScript fact guide!");
```
Comment (jdwilkin4) [inline frontend-cert/js-projects/js-chat-bot/script.js:1]: It looks like JSbot is introduced twice.
Once on this line and then again on line 7. 
So, I vote we update this line here

```suggestion
console.log("Hello! I am going to provide you with some JavaScript facts!");
```

File: frontend-cert/js-projects/js-chat-bot/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+1. You should log a greeting message to the user, including a welcome statement from the bot introducing itself.
+1. You should create variables to store the bot's name, location, and favorite language.
+1. You should log a message to the user that includes the bot's name, location, and favorite language. For example: `Hi! I'm {botName}, I live in {botLocation}, and my favorite language is {favoriteLanguage}.`
+1. You should create a variable using `let` and assign it a fact about the bot's favorite language.
+1. You should log a message that combines a string and the fact stored in the variable to the user.
+1. You should reassign the variable you created earlier to a new fact about the bot's favorite language.
+1. You should log a message that combines a string and the new fact stored in the variable to the user.
+1. You should reassign the variable again to a final fact about the bot's favorite language.
+1. You should log a message using a template literal `` (`) `` to combine a string and the new fact stored in the variable.
```
Comment (jdwilkin4) [inline frontend-cert/js-projects/js-chat-bot/user-stories.md:9]: campers will not know about template literals at this point.
only string concatenation with the `+` operator

**Review by jdwilkin4 — COMMENTED** [review]
I think this is off to a good start but I am not sure how the testing is going to go for this

This is the current setup for most of the console statements.

```js
console.log("My name is", botName, "and I live in", botLocation);
console.log("My favorite programming language is", favoriteLanguage);
```
but technically campers could do something like this, or some sort of variation of that

```js
console.log( "My name is " + botName + " and I live in " + botLocation)
```
but also, if we are asking them to come up with their own variable names, I am not sure how we would test for that in this situation. I guess with more regex 🤔 

With the other labs, we have the luxury of working with more like functions. So having them create their own variables is easier because we have more to work with that point.

But since this is the first one, the campers will know very little at this point which does kind of restrict us a little.

So, this might be the one and only time, where we are a little more strict on the structure and variables.

But I want to hear from @moT01 on what he thinks

**Review by moT01 — COMMENTED** [review]
Here's some adjustments that I think could be a little better:

```js
console.log("Hello! I'm your coding fun fact guide!");

const botName = "JsBot";
const botLocation = "planet Eris";
const favoriteLanguage = "JavaScript";

console.log("My name is " + botName + " and I live on " + botLocation + ".");
console.log("My favorite programming language is " + favoriteLanguage + ".");

let codingFact = "Did you know that " + favoriteLanguage + " was created in just 10 days?";

console.log(codingFact);

codingFact = "Another fun fact: " + favoriteLanguage + "was originally called Mocha!";

console.log(codingFact);

codingFact = "Also, " + favoriteLanguage + " is the most popular programming language in the world.";

console.log(codingFact);

console.log("It was fun sharing these facts with you. Goodbye! - " + botName + " from " + botLocation);
```

And these can be the user stories:

1. You should log `"Hello! I'm your coding fun fact guide!"` to the console as a greeting message to the user.
1. You should three `const` variables: `botName`, botLocation`, and `favoriteLanguage, to store the bot's name, where it's from, and it's favorite coding language, respectively.
1. You should use string concatenation to log `"My name is (botName) and I live on (botLocation)."` to the console.
1. You should use string concatenation to log `"My favorite programming language is (favoriteLanguage)."` to the console.
1. You should use `let` to create a `codingFact` variable and assign it a string that is a fun fact about your bot's favorite coding language and uses string concatenation to include the use of the `favoriteLanguage` variable.
1. You should log the `codingFact` to the console.
1. You should reassign the `codingFact` variable to a new fact about the bot's favorite language using the `favoriteLanguage` variable again.
1. You should log the `codingFact` to the console again.
1. You should reassign the `codingFact` variable again to another new fact about the bot's favorite language using the `favoriteLanguage` variable.
1. You should log the `codingFact` to the console a third time.
1. You should log `"It was fun sharing these facts with you. Goodbye! - (botName) from (botLocation)." to the console as a farewell statement from the bot.


The user stories a little more restrictive, but it gives us something to test against. It's just a suggestion, feel free to make adjustments or comment.

**Review by jdwilkin4 — APPROVED** [review]
LGTM 🚀

**Pull request merged**